src/dir.targets

```diff
@@ -35,12 +35,21 @@
   <Import Project="$(RepoRoot)\targets\OverrideGetReferenceAssemblyPaths.targets"
           Condition="'$(NetCoreBuild)' == 'true'"/>
 
-  <!-- OSS sign assemblies after compile runs. -->
-  <Import Condition="Exists('$(ToolsDir)\sign.targets')" Project="$(ToolsDir)\sign.targets" />
-  
+  <PropertyGroup Condition="'$(IsTestProject)' != 'true'">
```

Comment:
Satellite assembly signing probably also needs to be changed: [https://github.com/Microsoft/msbuild/blob/master/src/UpdateLocalizedResources.targets#L112-L140](https://github.com/Microsoft/msbuild/blob/master/src/UpdateLocalizedResources.targets#L112-L140)

Not sure how you open source sign an existing dll with Roslyn. If it does support signing dlls, then probably you can just delete the code and register the dlls with Roslyn. The CI does debug localized